test/index.js

```diff
@@ -3,6 +3,7 @@
 import test from 'tape';
 import irecord from '../index.js';
 import rx from './rx';
+import stampit from './stampit';
```

Comment:
How do you mean? It's importing test/stampit.js, just like you imported test/rx.js.